lib/locales/ta.ts

```diff
@@ -0,0 +1,998 @@
+import { Locale } from '../types/locale';
+
+export const locale: Locale = {
+  id: 'ta',
+
+  seasons: {
+    advent: {
+      season: 'திருவருகைக்காலம்',
+      weekday: '$t(weekdays:{{dow}}, capitalize) - $t(ordinals:{{week}}) திருவருகைக்கால வாரநாள்',
+      sunday: '$t(ordinals:{{week}}, capitalize) திருவருகைக்கால ஞாயிறு',
+      privileged_weekday: '$t(months:11, capitalize) {{day}}',
+    },
+
+    christmas_time: {
+      season: 'கிறிஸ்து பிறப்பு காலம்',
+      day: '$t(weekdays:{{dow}}, capitalize) கிறிஸ்து பிறப்பு காலம்',
+      octave: '{{count}}ஆம் நாள் கிறிஸ்து பிறப்பின் எண்கிழமை',
+      second_sunday_after_christmas: 'Second Sunday after the Nativity of the Lord',
+      before_epiphany: '$t(months:0, capitalize) {{day}}',
+      after_epiphany: '$t(weekdays:{{dow}}, capitalize) - திருக்காட்சிக்கு பின்',
+    },
+
+    ordinary_time: {
+      season: 'பொதுக்காலம்',
+      weekday: '$t(weekdays:{{dow}}, capitalize) - $t(ordinals:{{week}}) பொதுக்கால வாரநாள்',
+      sunday: '$t(ordinals:{{week}}, capitalize) பொதுக்கால ஞாயிறு',
+    },
+
+    lent: {
+      season: 'தவக்காலம்',
+      weekday: '$t(weekdays:{{dow}}, capitalize) - $t(ordinals:{{week}}) தவக்கால வாரநாள்',
+      sunday: '$t(ordinals:{{week}}, capitalize) தவக்கால ஞாயிறு',
+      day_after_ash_wed: '$t(weekdays:{{dow}}, capitalize) விபூதி புதனுக்குப் பின்',
+      holy_week_day: '$t(weekdays:{{dow}}, capitalize) - புனித வாரம்',
+    },
+
+    paschal_triduum: {
+      season: 'பாஸ்காவின் மூன்று புனித நாட்கள்',
+    },
+
+    easter_time: {
+      season: 'பாஸ்காக்காலம்',
+      weekday: '$t(weekdays:{{dow}}, capitalize) - $t(ordinals:{{week}}) பாஸ்காக்காலம் வாரநாள்',
+      sunday: '$t(ordinals:{{week}}, capitalize) பாஸ்காகாலம் ஞாயிறு',
+      octave: '$t(weekdays:{{dow}}, capitalize) பாஸ்கா எண்கிழமை',
+    },
+  },
+
+  periods: {
+    epiphany: 'திருக்காட்சி',
+    holy_week: 'புனித வாரம்',
+  },
+
+  ranks: {
+    solemnity: 'பெருவிழா',
+    sunday: 'ஞாயிறு',
+    feast: 'திருவிழா',
+    memorial: 'நினைவு',
+    // optional_memorial: 'optional memorial',
+    weekday: 'வாரநாள்',
+  },
+
+  cycles: {
+    proper_of_time: 'Proper of Time',
+    proper_of_saints: 'Proper of Saints',
+    sunday_year_a: 'ஆண்டு A',
+    sunday_year_b: 'ஆண்டு B',
+    sunday_year_c: 'ஆண்டு C',
+    weekday_year_1: 'Cycle I',
+    weekday_year_2: 'Cycle II',
+    psalter_week_1: 'Week I',
+    psalter_week_2: 'Week II',
+    psalter_week_3: 'Week III',
+    psalter_week_4: 'Week IV',
+  },
```

Comment:
@pvanthony, if you could take a look and suggest a translation for these keys as well, that would be great!